<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 170, 0);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>700</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
border-radius:5px;</string>
   </property>
   <widget class="QLineEdit" name="username_line">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>105</y>
      <width>310</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(200, 255, 199);
color: rgb(0, 0, 0);
border: 1px solid rgb(255, 255, 127);
padding-left:7px;
padding-bottom:2px;</string>
    </property>
    <property name="placeholderText">
     <string notr="true">Enter your username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_line">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>175</y>
      <width>310</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(200, 255, 199);
color: rgb(0, 0, 0);
border: 1px solid rgb(255, 255, 127);
padding-left:7px;
padding-bottom:2px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string notr="true">Enter your password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signup_button">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>245</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#signup_button{
background-color: rgb(0, 170, 0);
border: 1px solid rgb(0, 0, 0);
}
QPushButton#signup_button:pressed{
background-color: rgb(120, 170, 0);
}</string>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>55</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 85, 0)</string>
    </property>
    <property name="text">
     <string>î©Š</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_line">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>35</y>
      <width>591</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(200, 255, 199);
color: rgb(0, 0, 0);
border: 1px solid rgb(255, 255, 127);
padding-left:7px;
padding-bottom:2px;</string>
    </property>
    <property name="placeholderText">
     <string notr="true">Enter your email</string>
    </property>
   </widget>
   <widget class="QLabel" name="login_question">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>300</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>if you have an account:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_button">
    <property name="geometry">
     <rect>
      <x>234</x>
      <y>295</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#login_button{
background-color: rgb(0, 170, 0);
border: 1px solid rgb(0, 0, 0);
}
QPushButton#login_button:pressed{
background-color: rgb(120, 170, 0);
}</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
